lang/python/python-sentry-sdk/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2015-2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
```

Comment:
Minor nitpick: unless this makefile was available in 2015, I think this should be just 2019 :joy: